### PR TITLE
[BrokerLoad] Support read properties for broker load when read data

### LINKED ISSUE #5844: [Feature][BrokerLoad] Support read properties for BrokerLoad Statement
**Is your feature request related to a problem? Please describe.**
 Now, broker load job cannot set some read properties by broker load statement, and broker load on json format file would failed 

**Describe the solution you'd like**
Support read properties for BrokerLoad Statement and make broker load on json format file succeed

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by caiconghui**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have created an issue on (Fix #5844 ) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/analysis/DataDescription.java [hunk 1]
```diff
@@ -81,6 +81,14 @@
  */
 public class DataDescription {
     private static final Logger LOG = LogManager.getLogger(DataDescription.class);
+
+    private static final String LINE_DELIMITER = "line_delimiter";
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/DataDescription.java:85]: Could we use the definition in LoadStmt directly?

File: fe/fe-core/src/main/java/org/apache/doris/load/BrokerFileGroup.java [hunk 2]
```diff
@@ -237,7 +238,8 @@ public void parse(Database db, DataDescription dataDescription) throws DdlExcept
             jsonPaths = dataDescription.getJsonPaths();
             jsonRoot = dataDescription.getJsonRoot();
             fuzzyParse = dataDescription.isFuzzyParse();
-            readJsonByLine = dataDescription.isReadJsonByLine();
+            readJsonByLine = true;
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/load/BrokerFileGroup.java:241]: Add comment:
"For broker load, we only support reading json format data line by line, so we set `readJsonByLine` to true here."

File: fe/fe-core/src/test/java/org/apache/doris/analysis/DataDescriptionTest.java [hunk 3]
```diff
@@ -17,6 +17,7 @@
 
 package org.apache.doris.analysis;
 
+import com.google.common.collect.Maps;
```
Comment (morningman) [inline fe/fe-core/src/test/java/org/apache/doris/analysis/DataDescriptionTest.java:20]: Import order

File: fe/fe-core/src/main/java/org/apache/doris/analysis/DataDescription.java (hunk 1 above)
Comment (caiconghui) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/DataDescription.java]: done

File: fe/fe-core/src/test/java/org/apache/doris/analysis/DataDescriptionTest.java (hunk 3 above)
Comment (caiconghui) [reply fe/fe-core/src/test/java/org/apache/doris/analysis/DataDescriptionTest.java]: done

File: fe/fe-core/src/main/java/org/apache/doris/load/BrokerFileGroup.java (hunk 2 above)
Comment (caiconghui) [reply fe/fe-core/src/main/java/org/apache/doris/load/BrokerFileGroup.java:242]: done

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**